templates/mailhog/0/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  mailhog:
+    image: mailhog/mailhog:v1.0.0
+    environment:
+      MH_UI_WEB_PATH: ${mh_ui_web_path}
+      MH_SMTP_BIND_ADDR: ${mh_smtp_bind_addr}
```

Comment:
without any publish ports, how is this service accessible?

Reply:
The service is used internally as a mail proxy SMTP port 25. Containers use an external_link to reference the container (sample below). The Web UI is exposed via a Rancher Load Balancer (LB); we do not expose the service directly in our use case as we do hostname based routing in Rancher LB.

It should be pretty easy to make an option to "expose publicly" or something like that. I avoided the implementation since we often experience significant issues using many of the existing catalog services due to their propensity to expose on common ports (ex. 80).

Reply:
@RonWilliams, could you please add a simple `templates/mailhog/0/README.md` explaining it for user that would like to publish smtp and/or UI service (8025 tcp port) by rancher LB??

Reply:
@rawmind0 Good idea, added.